docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "System Table Configuration"
+date: 2024-01-23T16:30:21+05:30
+---
+
+## Configuration changes required for System Table Integration
+
+There is no unique configuration process for System Table ; however, the format for the **BELOW CONFIGURATIONS**
```

Comment:
This sentence is a bit hard to parse. I think we could do something close to: "The only change required to the configuration is the following: auditlogprefix_source_path - Instead of adding a fully qualified path (s3 or GC) for auditlog, add keyword system in this column. This will enable the system table integration. "

Reply:
done